veps/sig-network/hotpluggable-nad-ref.md

```diff
@@ -184,9 +186,12 @@ The following scenarios are to be added to the e2e tests:
 
 ## Graduation Requirements
 
+### Alpha
+- Alpha stage will be skipped as most of the required logic already exists.
```

Comment:
I think you need to be a bit more convincing.
Perhaps you can be more specific on why it is safe to do so. E.g. no API change is introduced, depends on 1, 2, 3 functionalities that already server a, b, c hotplug changes... etc.